lib/spack/spack/cmd/style.py

```diff
@@ -252,20 +263,41 @@ def run_mypy(file_list, args):
 
     rewrite_and_print_output(output, args)
 
-    if returncode == 0:
-        tty.msg("mypy checks were clean")
-    else:
-        tty.error("mypy checks found errors")
+    print_tool_result("mypy", returncode)
+    return returncode
+
+
+def run_isort(file_list, args):
+    isort_cmd = which("isort")
+    if isort_cmd is None:
+        tty.error("isort is not available in path, skipping")
+        return 1
+
+    print_tool_header("isort")
+    check_fix_args = () if args.fix else ("--check", "--diff")
+
+    pat = re.compile("ERROR: (.*) Imports are incorrectly sorted and/or formatted")
+    replacement = "ERROR: {0} Imports are incorrectly sorted and/or formatted"
```

Comment:
How stable is this error message across multiple versions of isort?

Reply:
changed it to a shorter version but that string has been there for a while -- even in the test suite.